extensions/azure-cosmos/runtime/src/main/java/io/quarkiverse/azure/cosmos/runtime/CosmosConfig.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkiverse.azure.cosmos.runtime;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "azure.cosmos", phase = ConfigPhase.RUN_TIME)
+public class CosmosConfig {
+
+    /**
+     * The flag to enable the cosmos. If set to false, the cosmos will be disabled
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean enabled;
+
+    /**
+     * The endpoint of Azure Cosmos DB. Required if quarkus.azure.cosmos.enabled is set to true
+     */
+    @ConfigItem
+    public Optional<String> endpoint;
```

Comment:
Do we support multiple databases and containers?

Reply:
We support user to specify one endpoint of Azure Cosmos DB account, and the extension exposes a `CosmosClient` or `CosmosAsyncClient`. User can use the client to create multiple databases/containers per their needs.